etc/dehydrated/hook.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+CONSUL_ROOT="http://localhost:8500/v1/kv/nginx"
```

Comment:
The ContainerPilot configuration supports non-localhost Consul paths [depending on the value of `CONSUL_AGENT` variable](https://github.com/autopilotpattern/nginx/blob/master/etc/containerpilot.json#L2). We should probably try to translate the same configuration to bash here:

```
"consul": "{{ if .CONSUL_AGENT }}localhost{{ else }}{{ .CONSUL }}{{ end }}:8500",
```